pipelines/ubuntu-jammy/pipeline.yml

```diff
@@ -536,15 +537,10 @@ jobs:
       AWS_ACCESS_KEY_ID: ((hmac_accesskey))
       AWS_SECRET_ACCESS_KEY: ((hmac_secret))
       COMMIT_PREFIX: candidate
-      COPY_KEYS: |
-        aws/bosh-stemcell-%s-aws-xen-hvm-ubuntu-(@= data.values.stemcell_details.os @)-go_agent.tgz
-        google/bosh-stemcell-%s-google-kvm-ubuntu-(@= data.values.stemcell_details.os @)-go_agent.tgz
-        openstack/bosh-stemcell-%s-openstack-kvm-ubuntu-(@= data.values.stemcell_details.os @)-go_agent.tgz
-        openstack/bosh-stemcell-%s-openstack-kvm-ubuntu-(@= data.values.stemcell_details.os @)-go_agent-raw.tgz
-        warden/bosh-stemcell-%s-warden-boshlite-ubuntu-(@= data.values.stemcell_details.os @)-go_agent.tgz
-        vsphere/bosh-stemcell-%s-vsphere-esxi-ubuntu-(@= data.values.stemcell_details.os @)-go_agent.tgz
-        vcloud/bosh-stemcell-%s-vcloud-esxi-ubuntu-(@= data.values.stemcell_details.os @)-go_agent.tgz
-        azure/bosh-stemcell-%s-azure-hyperv-ubuntu-(@= data.values.stemcell_details.os @)-go_agent.tgz
+      COPY_KEYS:
+      #@ for iaas in data.values.stemcell_details.include_iaas:
+        - (@= iaas.iaas @)/bosh-stemcell-%s-(@= iaas.iaas @)-(@= iaas.hypervisor @)-(@= data.values.stemcell_details.os_name @)-go_agent.tgz
```

Comment:
Wouldn't using [join](https://github.com/google/starlark-go/blob/master/doc/spec.md#stringjoin) preserve the multi-line string format?